Duckling/Numeral/HI/Corpus.hs

```diff
@@ -61,40 +57,28 @@ allExamples = concat
   , examples (NumeralValue 10)
              [ "दस"
              ]
---  , examples (NumeralValue 15)
```

Comment:
let's get these examples working if they are valid and add them under the existing ones below